### PR TITLE
Conversion to NET Standard 2.0

### PR DESCRIPTION
**Pull request by warwick-pote**
Conversion of the StatsdClient library to .NET Standard 2.0 so that it can be reference for use by either .NET Framework (4.6.1) or .NET Core applications.

### CODE REVIEW

File: StatsdClient/StatsdClient.nuspec [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <metadata>
     <id>StatsdCsharpClient</id>
     <version>$version$</version>
-    <title>Simple Statsd Client for .Net 4.5</title>
+    <title>Simple Statsd Client for .Net 4.6.1</title>
```
Comment (niemyjski) [inline StatsdClient/StatsdClient.nuspec]: Can we just call this Simple StatsD Client

File: StatsdClientTests/StatsdClientTests.csproj [hunk 2]
```diff
@@ -8,14 +8,15 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>StatsdClientTests</RootNamespace>
     <AssemblyName>StatsdClientTests</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```
Comment (niemyjski) [inline StatsdClientTests/StatsdClientTests.csproj]: Can we update this to run on .net core 2.0 (xplat) and ditch configuration?

File: StatsdClient/StatsdClient.nuspec (hunk 1 above)
Comment (warwick-pote) [reply StatsdClient/StatsdClient.nuspec]: Can't see why not. I will make the change in the next few days.

File: StatsdClientTests/StatsdClientTests.csproj (hunk 2 above)
Comment (warwick-pote) [reply StatsdClientTests/StatsdClientTests.csproj]: Sure. I will update the project and commit it along with the previous request.

**Comment by niemyjski** [comment]
Thanks for the PR! Can you please try reviewing the other pr as well and hopefully we can get this out with more changes.

**Comment by warwick-pote** [comment]
You are welcome.

I took a brief look at the other pr and these are my thoughts.
a) It was done is 2015 and so was based off older revision of the master.
b) I don't believe the unit tests, need to be refactored as they appear to be working. But maybe I am missing something that Jeff spotted.

Basically this means that the only class that needs/could to be merged in would be Statsd.cs. However there are a number of conflicts here that mean that this would need to be a manual merge.

In my opinion, it would be better for the author to start over with a clean version of the source.

**Pull request merged**